R/write_xlsx.R

```diff
@@ -358,6 +359,7 @@ write_xlsx <- function(x, file, as_table = FALSE, ...) {
         na.strings  = na.strings
       )
     } else {
+      # TODO add_data()?
```

Comment:
Ah, at first I was quite confused, but yes, we might look into this! The problem is mainly, I still don't really use this function, hence the overlooked fixes.

Reply:
But this can wait until the next release. I'm not losing sleep over this :)